project4/project/ebs/templates/ebs/manage_blog.html

```diff
@@ -8,7 +8,62 @@
 	<div id="message"></div>
 {% endfor %}
 <div class="container">
-	<a class="pull-right btn btn1 top-margin" href="{% url 'create_blog' %}">Add Blog</a> 
+	<div class="table-responsive">
+  		<table class="table results">
+			<thead>
+				<tr>
+					<th colspan="2">
+						<input type="checkbox" id="select-all">
+						<a  id="delete-button" class=" btn btn2">
+  						<i class="fa fa-trash-o fa-lg"></i>   Delete</a>
+  						
+					</th>
+					<th>
+						<div class="input-group search">
+						<input type="text"class=" search1 form-control" placeholder="Search">
```

Comment:
search1 class is not as per CSS naming conventions and give spacing in appropriate places.